### ISSUE TITLE
Meta.getTables() does not include PostgreSQL foreign data wrapper tables

### ISSUE THREAD

**Issue by lukaseder** [jOOQ/jOOQ#11255]
When using a foreign data wrapper table in PostgreSQL:

```sql
create extension if not exists postgres_fdw;
create server loopback foreign data wrapper postgres_fdw;
create user mapping for postgres server loopback options (user 'postgres', password 'test');
create foreign table t_fdw (id int, title text) server loopback options (table_name 't_book');
```

The `t_fdw` table is not retrieved `Meta.getTables()`

**Comment by lukaseder** [comment]
The `table_type` is `FOREIGN TABLE`:

```
+---------+-----------+----------+-------------+
|table_cat|table_schem|table_name|table_type   |
+---------+-----------+----------+-------------+
|{null}   |public     |t_fdw     |FOREIGN TABLE|
+---------+-----------+----------+-------------+
```

And we're currently not including that:

```java
case POSTGRES:
    types = new String[] { "TABLE", "VIEW", "SYSTEM_TABLE", "SYSTEM_VIEW", "MATERIALIZED VIEW" };
```

**Comment by lukaseder** [comment]
Fixed in jOOQ 3.15.0 and 3.14.5 (#11257)
